### PR TITLE
Create 2017-11-16-tenser's-transformation.markdown

### PR DESCRIPTION
**Pull request by Eikrem63**

### CODE REVIEW

File: _posts/2017-11-16-tenser's-transformation.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+**Components**: V, S, M (a few hairs from a bull)
+
+**Duration**: Concentration, up to 10 minutes
+
+You endow yourself with endurance and martial prowess fueled by magic. Until the spell ends, you can’t cast spells, and you gain the following benefits: 
+
+- You gain 50 temporary hit points. If any of these remain when the spell ends, they are lost. 
+- You have advantage on attack rolls that you make with simple and martial weapons. 
+- When you hit a target with a weapon attack, that tar- ; i get takes an extra 2d 12 force damage.
```
Comment (Traneptora) [inline _posts/2017-11-16-tenser's-transformation.markdown]: Now this is an interesting typo. I don't know how it happened, but it should say "that target takes"

Comment (Traneptora) [reply _posts/2017-11-16-tenser's-transformation.markdown]: Also, `2d12` does not have a space in it.

File: _posts/2017-11-16-tenser's-transformation.markdown [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+**Components**: V, S, M (a few hairs from a bull)
+
+**Duration**: Concentration, up to 10 minutes
+
+You endow yourself with endurance and martial prowess fueled by magic. Until the spell ends, you can’t cast spells, and you gain the following benefits: 
+
+- You gain 50 temporary hit points. If any of these remain when the spell ends, they are lost. 
+- You have advantage on attack rolls that you make with simple and martial weapons. 
+- When you hit a target with a weapon attack, that tar- ; i get takes an extra 2d 12 force damage. 
+- You have proficiency with all armor, shields, simple i] weapons, and martial weapons.
```
Comment (Traneptora) [inline _posts/2017-11-16-tenser's-transformation.markdown]: There's a random `i]` here for some reason.

File: _posts/2017-11-16-tenser's-transformation.markdown (hunk 1 above)
Comment (Eikrem63) [reply _posts/2017-11-16-tenser's-transformation.markdown]: copy and paste gives some very strange errors sometimes.

**Pull request merged**